kvmagent/kvmagent/test/localstorage_testsuite/test_file_metadata.py

```
@@ -0,0 +1,797 @@
+        vm_uuid = 'f3' * 16
+        meta_path = self._meta_path(vm_uuid)
+        with open(meta_path, 'w') as f:
+            f.write('data')
+        os.chmod(self.tmpdir, 0o444)
+        try:
+            with self.assertRaises(Exception) as ctx:
+                self.handler._do_cleanup(meta_path)
+            self.assertIn('failed to cleanup metadata file',
+                          str(ctx.exception))
+        finally:
+            os.chmod(self.tmpdir, 0o755)
```

Ruff (0.15.7)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)